easybuild/easyconfigs/c/CheckM/CheckM-1.0.18-foss-2019a-Python-2.7.15.eb

```diff
@@ -0,0 +1,45 @@
+# Updated from previous config
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'PythonPackage'
+
+name = 'CheckM'
+version = '1.0.18'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/Ecogenomics/CheckM'
+description = """CheckM provides a set of tools for assessing the quality of genomes recovered from isolates,
+ single cells, or metagenomes."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = ['https://pypi.python.org/packages/source/c/checkm-genome']
+sources = ['checkm-genome-%(version)s.tar.gz']
+checksums = ['0dcf31eab5e340a0fff37d7a5091d46d9269b0708db8f789adcd7cbd2a09a2b7']
+
+options = {"modulename": "checkm"}
```

Comment:
@Darkless012 You can drop this, since by default `PythonPackage` will use the lowercase software name already